src/platform/user/selectors.js

```diff
@@ -12,11 +13,27 @@ export const isLOA1 = state => selectProfile(state).loa.current === 1;
 export const isMultifactorEnabled = state => selectProfile(state).multifactor;
 export const selectAvailableServices = state => selectProfile(state)?.services;
 export const selectPatientFacilities = state =>
-  selectProfile(state)?.facilities?.map(({ facilityId, isCerner }) => ({
-    facilityId,
-    // This overrides the MPI isCerner flag in favor of the list maintained in vets-website.
-    isCerner: isCerner || CERNER_FACILITY_IDS.includes(facilityId),
-  })) || null;
+  selectProfile(state)?.facilities?.map(({ facilityId, isCerner }) => {
+    // Derive if the user belongs to a Cerner facility in the FE maintained list.
+    const hasCernerFacilityID = CERNER_FACILITY_IDS.includes(facilityId);
+
+    // Derive if the feature toggle is on.
+    const showNewScheduleViewAppointmentsPage =
+      state?.featureToggles?.[
+        featureFlagNames.showNewScheduleViewAppointmentsPage
+      ];
+
+    // Derive if they are a 200CRNR Cerner patient.
+    const isCernerPatient = state?.user?.profile?.isCernerPatient;
```

Comment:
You could use the selectProfile selector here. And the feature toggles one above. It reduces the risk that someone tweaks the redux state and breaks code not using the expected selectors. Not a big thing, though

Reply:
Updated! Thanks! 🙂

Reply:
I was worried we were using reselect everywhere for all these selectors, which can cause some serious perf issues when misused, but now just seeing that these are just helper funcs I can see the value in them 👍